src/items/items.service.ts

```diff
@@ -31,6 +33,28 @@ export class ItemsService {
     return await this.itemsModel.find({ user_uuid });
   }
 
+  async findInOrder(itemPaginationInput: ItemPaginationInput): Promise<ItemPagination> {
+    const decode = (str: string):string => Buffer.from(str, 'base64').toString('binary');
+    const encode = (str: string):string => Buffer.from(str, 'binary').toString('base64');
+
+    const quantity = itemPaginationInput.quantity;
+    let items, lastElementAttribute = "";
+    if (itemPaginationInput.cursor_type === 'Date') {
```

Comment:
I would suggest that in the input the cursor_type should just be equal to the name of the field in this case "updated_at". And then instead of doing "this.itemsModel.find({ updated_at: ..." we do 
"this.itemsModel.find({ [itemPaginationInput.cursor_type]: ..."

This way we can avoid adding if statement for every new sorting option

Reply:
aight FINE